vault/core.go

```diff
@@ -131,6 +131,19 @@ type unlockInformation struct {
 	Nonce string
 }
 
+type counters struct {
```

Comment:
Could we move this struct into the counters file?

Reply:
Will do.